cmd/tsidp/tsidp.go

```diff
@@ -305,6 +349,7 @@ type idpServer struct {
 	funnel      bool
 	localTSMode bool
 	rootPath    string // root path, used for storing state files
+	strictOAuth bool   // enable strict OAuth mode
```

Comment:
consider keeping this name symmetric with the flag, this particular name has a very non-specific meaning, but the current use is very specific. Any reason not to make it more specific like allowInsecureRegistration ? (or disallowInsecureRegistration if you want this truthiness)